tools/cli/pkg/command/orchestration.go

```diff
@@ -125,6 +125,7 @@ Global Account ID:  {{.GlobalAccountID}}
 Subaccount ID:      {{.SubAccountID}}
 Runtime ID:         {{.RuntimeID}}
 Shoot Name:         {{.ShootName}}
+Instance ID:        {{.InstanceID}}
```

Comment:
the operation response object won't contain the instance ID, this line is not needed.